src/pcapi/core/offerers/validation.py

```diff
@@ -46,6 +72,11 @@ def check_venue_edition(modifications, venue):
         if venue_with_same_siret:
             errors = ApiErrors()
             errors.add_error("siret", "Un lieu avec le même siret existe déjà")
+            raise errors
+    if None in venue_disability_compliance and None in modifications_disability_compliance:
+        errors = ApiErrors()
+        errors.add_error("global", "L'accessibilité du lieu doit etre définit.")
```

Comment:
```suggestion
        errors.add_error("global", "L'accessibilité du lieu doit etre définie.")
```
La typo n'a pas été corrigée.